linesearch.go

```diff
@@ -63,46 +60,34 @@ func (ls *LinesearchMethod) Iterate(loc *Location) (Operation, error) {
 		// Init was not called. What to do? What about ls's internal state?
 
 	case MajorIteration:
-		// We previously requested MajorIteration but since we're here, the
+		// We previously commanded MajorIteration but since we're here, the
```

Comment:
"The previous updated location did not converge the full optimization. Initialize a new Linesearch."